presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestExtractSpatialInnerJoin.java

```diff
@@ -80,24 +109,33 @@ public void testDoesNotFire()
         // SphericalGeography operand
         assertRuleApplication()
                 .on(p ->
-                        p.filter(PlanBuilder.expression("ST_Distance(a, b) < 5"),
+                        p.filter(
+                                sqlToRowExpression(
+                                        "ST_Distance(a, b) < 5",
+                                        ImmutableMap.of("a", SPHERICAL_GEOGRAPHY, "b", SPHERICAL_GEOGRAPHY)),
                                 p.join(INNER,
                                         p.values(p.variable("a", SPHERICAL_GEOGRAPHY)),
                                         p.values(p.variable("b", SPHERICAL_GEOGRAPHY)))))
                 .doesNotFire();
 
-        assertRuleApplication()
-                .on(p ->
-                        p.filter(PlanBuilder.expression("ST_Contains(polygon, point)"),
-                                p.join(INNER,
-                                        p.values(p.variable("polygon", SPHERICAL_GEOGRAPHY)),
-                                        p.values(p.variable("point", SPHERICAL_GEOGRAPHY)))))
-                .doesNotFire();
+//        assertRuleApplication()
+//                .on(p ->
+//                        p.filter(
+//                                sqlToRowExpression(
+//                                        "ST_Contains(polygon, point)",
```

Comment:
@jagill, `ST_Contains(SPHERICAL_GEOGRAPHY, SPHERICAL_GEOGRAPHY)` is not defined in `GeoFunctions`. This signature doesn't seem to be valid. Do you happen to now why?

Reply:
We haven't implemented `ST_Contains` for spherical geometries yet.  Our spherical geometry support is not yet complete :(.  There is an outstanding PR for the polygon-containing-point option (#13133), but it seems stalled.

Reply:
Moved to individual disabled tests with a TODO until we land that change.